charmcraft/commands/store/store.py

```diff
@@ -134,3 +134,9 @@ def list_revisions(self, name):
                 errors=errors,
             ))
         return result
+
+    def release(self, name, revision, channels):
+        """Release one or more revisions for a package."""
+        endpoint = '/v1/charm/{}/releases'.format(name)
+        items = [{'revision': revision, 'channel': channel} for channel in channels]
+        self._client.post(endpoint, items)
```

Comment:
Interesting that the API supports arbitrary revisions to arbitrary channels.
Is this for the "release all the different Architectures simultaneously" support?
Do we have a design for how we might do that in our CLI?
(AIUI, this is around 'i want to release the Juju 2.8.1 snap, but that involves revision 1234 for amd64, 1235 for arm64, 1239 for ppc64le, etc.)

Reply:
yes, that's for this. We're yet to discuss how to do it in the CLI, but it got messy so I suggested we move ahead without it and revisit when we know more

Reply:
Yeap, specially in the "not only arch but also OS and series" world.